packages/babel-plugin-transform-typescript/src/index.js

```diff
@@ -44,11 +44,90 @@ function registerGlobalType(programScope, name) {
 }
 
 export default declare(
-  (api, { jsxPragma = "React", allowNamespaces = false }) => {
+  (
+    api,
+    { jsxPragma = "React", allowNamespaces = false, declareFields = false },
```

Comment:
Should we name this `allowDeclareFields` (or `allowDeclare`) to match `allowNamespaces`?

Reply:
:+1: `allowDeclareFields`